{
  "_annotation": {
    "cve_id": "CVE-2024-42367",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-42367.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:aiohttp:aiohttp:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "63AE4611-EB42-59BF-B192-A33B734195CF",
                "versionEndExcluding": "3.10.2",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}